syntax = "proto3";

package Error;

enum ErrorCode {
	OK = 0;
	FAILED = 1;
}

message CommonError {
  ErrorCode Code = 1;
  string Reason = 2;
}
